src/pudl/metadata/fields.py

```diff
@@ -4736,24 +4814,36 @@
         "opex_total": {"description": "Overall expenses for the transmission line."},
     },
     "out_ferc714__hourly_planning_area_demand": {
-        "timezone": {
-            "constraints": {
-                "enum": [
-                    "America/New_York",
-                    "America/Chicago",
-                    "America/Denver",
-                    "America/Los_Angeles",
-                    "America/Anchorage",
-                    "Pacific/Honolulu",
-                ]
-            }
-        },
+        "timezone": {"constraints": {"enum": US_TIMEZONES}},
         "report_date": {
             "constraints": {
                 "required": True,
             }
         },
     },
+    "core_eia930__hourly_balancing_authority_net_generation": {
+        "datetime_utc": {
+            "description": "Timestamp at the end of the hour for which the data is reported."
+        },
+    },
+    "core_eia930__hourly_balancing_authority_demand": {
+        "datetime_utc": {
+            "description": "Timestamp at the end of the hour for which the data is reported."
+        },
+    },
+    "core_eia930__hourly_balancing_authority_subregion_demand": {
+        "datetime_utc": {
+            "description": "Timestamp at the end of the hour for which the data is reported."
+        },
+        "demand_reported_mwh": {
+            "description": "Originally reported electricity demand for the balancing area subregion. Method of calculation may vary by BA.",
```

Comment:
Internally the BAs have different ways of calculating "demand" as generation minus interchange, because some of them use physical metered values, and others use virtual transactions that "move" generation economically but not physically. So the way they're allocating demand within their boundaries isn't necessarily the same for different BAs.